### PR TITLE
Tests for the backend game

### PR DESCRIPTION
**Pull request by mitchellolsthoorn**

### CODE REVIEW

**Comment by mitchellolsthoorn** [comment]
Still working on this

**Comment by mitchellolsthoorn** [comment]
It's finished now

**Comment by mitchellolsthoorn** [comment]
@newnottakenname Can you check the code and the tests

File: backend/src/main/nl/tudelft/ti2806/riverrush/game/Game.java [hunk 1]
```diff
@@ -185,4 +139,13 @@ public void collideAnimal(final Integer animal, final Integer team) {
     public Event getStateEvent() {
         return this.gameState.getStateEvent();
     }
+
+
+    public GameState getGameState() {
```
Comment (newnottakenname) [inline backend/src/main/nl/tudelft/ti2806/riverrush/game/Game.java]: Should not exist

File: backend/src/test/nl/tudelft/ti2806/riverrush/game/GameTest.java [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+        this.game = injector.getInstance(Game.class);
+
+        when(this.animal.getId()).thenReturn(ANIMAL_ID);
+        when(this.animal.getTeamId()).thenReturn(TEAM_ID);
+        when(this.track.addAnimal(TEAM_ID, this.animal)).thenReturn(TEAM_ID);
+
+    }
+
+    @Test
+    public void testStart() throws Exception {
```
Comment (newnottakenname) [inline backend/src/test/nl/tudelft/ti2806/riverrush/game/GameTest.java]: Should all be in the class made for them

**Pull request merged**